packages/react-vapor/src/components/select/hoc/tests/MultiSelectWithFilter.spec.tsx

```diff
@@ -1,88 +1,73 @@
-import {mount, ReactWrapper} from 'enzyme';
+import {render, screen} from '@test-utils';
 import * as React from 'react';
-import {Provider} from 'react-redux';
-import {Store} from 'redux';
 
-import {IReactVaporState} from '../../../../ReactVaporState';
-import {clearState} from '../../../../utils/ReduxUtils';
-import {TestUtils} from '../../../../utils/tests/TestUtils';
-import {filterThrough} from '../../../filterBox/FilterBoxActions';
-import {ItemBox} from '../../../itemBox/ItemBox';
-import {toggleSelect} from '../../SelectActions';
-import {ISelectOwnProps, SelectConnected} from '../../SelectConnected';
 import {MultiSelectWithFilter} from '../SelectComponents';
-import {ISelectWithFilterOwnProps} from '../SelectWithFilter';
 
 describe('Select', () => {
     describe('MultiSelectWithFilter', () => {
-        let wrapper: ReactWrapper<any, any>;
-        let store: Store<IReactVaporState>;
-
         const id: string = 'multi-select-with-filter';
 
-        const mountMultiSelect = (props: Partial<ISelectOwnProps & ISelectWithFilterOwnProps> = {items: []}) => {
-            wrapper = mount(
-                <Provider store={store}>
-                    <MultiSelectWithFilter id={id} {...props} />
-                </Provider>,
-                {attachTo: document.getElementById('App')}
-            );
-
-            store.dispatch(toggleSelect(id, true));
-            wrapper.update();
-        };
-
-        beforeEach(() => {
-            store = TestUtils.buildStore();
-        });
-
-        afterEach(() => {
-            store.dispatch(clearState());
-            wrapper?.unmount();
-        });
-
-        beforeAll(() => {
-            TestUtils.makeDebounceStatic();
-        });
-
         it('should add a duplicate if the filterValue is already selected', () => {
             const filterValue: string = 'a';
             const duplicateText: string = 'duplicate';
 
-            mountMultiSelect({items: [{value: filterValue, selected: true}], duplicateText});
-            store.dispatch(filterThrough(id, filterValue));
+            render(
+                <MultiSelectWithFilter
+                    id={id}
+                    duplicateText={duplicateText}
+                    items={[
+                        {value: filterValue, selected: true},
+                        {value: filterValue, selected: true},
+                    ]}
+                />,
+                {
+                    initialState: {},
+                }

```

Comment:
You don't need to send this object if the initialState is empty

Same comment for the other tests